python_file.py

```diff
@@ -0,0 +1,7 @@
+a=5
```

Comment:
Write a function to add integer numbers. The function should take any number of integers as input. Make use of *args